libraries/streaming-e2e-tests/react-app/.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
```

Comment:
For `botframework-streaming` it's possible to get the latest version from npm using the `next` tag, i.e:

`npm i botframework-streaming@next`

Which right now should resolve to https://www.npmjs.com/package/botframework-streaming/v/4.11.0-dev.20200911.2b4d430

Reply:
Ok. I'll rework it so that we use `next` in the configure to latest yaml, and that should eliminate the need for a myget access token and that would be 1 less variable needed for a user to create a pipeline off of `streaming-e2e2-tests-build.yml`  too 👍

Reply:
Done!

`get-latest-daily-ver.ps1` now uses npm install "package name"@next to install latest instead of myget
need for extra myget access token removed 🎊